### ISSUE TITLE
Rendering issue with dropdown in BarEnd

### ISSUE THREAD

**Issue by toddlang** [stsrki/Blazorise#51]
```
...
           <BarEnd>
                    <BarDropdown>
                        <BarDropdownToggle>Test</BarDropdownToggle>
                        <BarDropdownMenu>
                            <BarDropdownItem>Profile</BarDropdownItem>
                            <BarDropdownItem>Log Out</BarDropdownItem>
                        </BarDropdownMenu>
                    </BarDropdown>
            </BarEnd>
        </BarMenu>
```

When I activate the drop down, the menu itself renders at the left-edge of the screen rather than under the drop down that opened the menu.

**Comment by stsrki** [comment]
Thanks for the report.

**Comment by stsrki** [comment]
There are actually two issues since dropdown is placed in the `<BarEnd>` component:

1. You should place `BarDropdown` inside of `BarItem` component. That way the menu will be rendered bellow the button but it will possibly go beyond the page boundaries.
2. There is no `IsRightAligned` attribute on the `BarDropdownMenu`. I have fixed this and it will be available in the v0.6.1, possibly today or tomorrow.

**Comment by toddlang** [comment]
Item 1 was 95% of the fix I needed.  Sorry about that.
